code/modules/medical/cloning.dm

```diff
@@ -429,6 +429,26 @@
 	occupant.forceMove(exit)
 	icon_state = "pod_0"
 	eject_wait = FALSE //If it's still set somehow.
+	//do early ejection damage
+	var/completion = 10*((occupant.health + 100) / (heal_level + 100)) //same way completion is calculated for examine text, but out of 10 instead of 100
+	var/damage_rolls = 10 - round(completion) - (round(resource_efficiency) - 1) // 1 roll for each 10% missing, each improved pair of manipulators reduces one roll
+	var/hits = 0
+	while(damage_rolls > 0)
+		if(prob(25))//each roll has a 25% chance to give the occupant a bad time
+			hits++
+		damage_rolls--
+	//apply the damage
+	var/mob/living/carbon/human/H = occupant
+	while(hits>0)
+		if (hits>=4)
+			qdel(pick(H.internal_organs - H.internal_organs_by_name["brain"]))
+			hits -= 4
+		else
+			H.organs_by_name[pick(H.organs_by_name)].droplimb(override = 1, no_explode = 1, spawn_limb = 1, display_message = FALSE)
```

Comment:
this can pick the head or torso, at a minimum the torso needs to not be droplimbed

Reply:
Exclude the groin as well

Reply:
Torso and groin do not droplimb, if this pick() lands on them, nothing happens. Relevant snippet from from organ_external.dm:

```
/datum/organ/external/proc/droplimb(formal params)
	if(body_part == (UPPER_TORSO || LOWER_TORSO))
		return
```

Worth adding a comment to note the behavior

Reply:
comment added, also changed the PR body to match current version's behavior